menu/menu.yaml

```diff
@@ -381,11 +381,11 @@
       Title: Microsoft Generic Host
     - Url: nservicebus/hosting/web-application
       Title: Web Application
-    - Url: nservicebus/hosting/azure-functions
-      Title: Azure Functions
+    - Url: nservicebus/hosting/azure-functions-service-bus
+      Title: Azure Functions ServiceBus
```

Comment:
this should probably use `Service Bus`, not `ServiceBus`.

Maybe `Azure Functions (Service Bus)` or `Azure Functions for Service Bus`?

Reply:
```suggestion
      Title: Azure Functions for Service Bus
```